lib/gps.js

```diff
@@ -432,6 +562,40 @@ GPS.prototype.parseNmeaSentence = function(sentence) {
 
 };
 
+/*
+ * SkyTraq Binary Message Protocol
+ * http://cdn.sparkfun.com/datasheets/Sensors/GPS/Venus/638/doc/AN0003_v1.4.19.pdf
+ */
+GPS.prototype.parseSkyTraqSentence = function(sentence) {
```

Comment:
Those methods will only be called on that specific model due to a broken out 'listen' handle for this chipset, and chipset specific commands. Is the issue just with having the utility functions available?